src/routes/exercise/controllers.ts

```diff
@@ -21,19 +22,31 @@ const getAllExercises = async (req: Request, res: Response) => {
 };
 
 const createExercise = async (req: Request, res: Response) => {
-  // TO-DO: Middleware to validate the data and structure the info.
-  const createdExercise = await prisma.exercise.create({
-    data: req.body,
-    include: {
-      links: true,
-    },
-  });
+  try {
+    const { error, value } = exerciseSchema.validate(req.body);
```

Comment:
Instead of doing this, we can use the middleware to be more organized and readable. WDYT?

Reply:
yes , make sense but, could you be more specific here? do you want me to remove this validation from the controlled and add it as a middleware on the route? 

`router.post('/', validateExercise, controllers.createExercise)`

Reply:
Exactly, that's the idea. WDYT?